content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md

```diff
@@ -106,6 +106,26 @@ The following dependencies have been upgraded to address critical and high-sever
 
 When a state is consumed by a transaction, Corda now adds the ID of the consuming transaction in the `consuming_tx_id` column of the `vault_state` table. Corda only updates this database column for new transactions; for existing consumed states already in the ledger, the value of `consuming_tx_id` is null.
 
+### Node configuration change for better performance
+
+To reduce flow latency and improve throughput, we have changed the following defaults in the node configuration:
+* `enterpriseConfiguration.tuning.brokerConnectionTtlCheckIntervalMs` changed from 20 to 1 millisecond.
+* `enterpriseConfiguration.tuning.journalBufferTimeout` changed from 3333333 nanoseconds to 1000000 nanoseconds.
+* `notary.extraConfig.batchTimeoutMs` changes from 200 to 1.
+
+### DJVM removal
+
+With DJVM present, updates to Corda core must have also been compatible with
+the `core-deterministic` module. The following changes mitigate this issue:
+* The experimental component DJVM has been removed from this and all future releases.
+* Because of the DJVM removal, the `DriverParameters` class has changed. The two constructor parameters `djvmBootstrapSource`
+  and `djvmCordaSource` have been removed from `DriverParameters`. Any client code using `DriverParameters` now needs at least recompiling.
+
+### Additional signature verification
+
+Stricter signature verification when using public `ServiceHub` API function `recordTransactions()`. For more information,
```

Comment:
Think we're missing something here..

"Stricter signature verification **is now applied** when using public `ServiceHub` API function `recordTransactions()`." ??

Reply:
Rephrased